Create JSONMapper.java

zopeaj opened the pull request:
Creating a JSONMapper for reading and writing json data to files or outstream

zopeaj committed:
Create JSONMapper.java

JSONMapper file created

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/alibaba/fastjson?pullRequest=4391) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/alibaba/fastjson?pullRequest=4391) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/alibaba/fastjson?pullRequest=4391) it.</sub>

zopeaj committed:
Update JSONMapper.java

creating a json mapper extension for reading and writing of json data to files or from files

zopeaj committed:
Merge pull request #1 from zopeaj/zopeaj-jsonmapper

Update JSONMapper.java

zopeaj committed:
Update JSONMapper.java

Update  JSONMapper file
